lib/performance_test.rb

```diff
@@ -216,6 +216,7 @@ def write_report(fillers)
     fillers.each do |filler|
       filler.call(@rep_result)
     end
+    @rep_result.add_parameter('host', `hostname`)
```

Comment:
This code is executed on the test runner / controller. Probably need to get the hostnames from the VespaModel and we can have more than one host.

Reply:
Hmm ... need to change performance data query to use "matches" to support multiple values of a parameter, but can do.

Reply:
Meeeh, that may break other usages, though.

Reply:
@baldersheim what tests really use several nodes?

Reply:
Nvm, can make it work without breaking the others.